src/main/java/com/minecolonies/coremod/colony/jobs/JobDeliveryman.java

```diff
@@ -328,4 +356,87 @@ public boolean isActive()
     {
         return this.active;
     }
+
+    /**
+     * Check if the dman has the same destination request.
+     * @param request the incoming request.
+     * @return 0 if so, and 1 if not.
+     */
+    public int hasSameDestinationDelivery(@NotNull final IRequest<? extends Delivery> request)
```

Comment:
we're not in c++ :D make it a boolean return

Reply:
That is on purpose for the comparable